docs/operations-guide/java-versions.md

```diff
@@ -1,3 +1,25 @@
 # Java Versions
 
-Metabase will run on Java version 8 and 11.
+Metabase requires a Java Runtime Environment (JRE).
+
+We recommend the latest LTS version of JRE from [AdoptOpenJDK](https://adoptopenjdk.net/releases.html) with HotSpot JVM and x64 architecture, but any [supported version](https://adoptopenjdk.net/support.html) should work, including other types of JVM and architecture, and other distributions such as [OpenJDK](https://openjdk.java.net/), [Amazon Corretto](https://aws.amazon.com/corretto/) and [Oracle Java](https://www.java.com/).
+
+**Note** When using a "headless" version, the JVM needs to have AWT classes, which are sometimes not included. Otherwise Pulses and other functionality might not work correctly or not at all.
+
+When developing and building Metabase a Java Development Kit (JDK) is required. It is recommended to use the latest LTS version of JDK from AdoptOpenJDK with HotSpot JVM.
```

Comment:
Isn't this paragraph redundant? The paragraph above the note already basically said this. If this is specific to developing and building, then I recommend changing the first paragraph of this section like so:

"Metabase requires a Java Runtime Environment (JRE) to run, develop, or build it."

Reply:
JRE to run Metabase, but JDK to develop/build. It doesn't help, when we're recommending AdoptOpenJDK's JRE - too many acronyms, but the devil's in the details ;-)